global.json

```diff
@@ -1,13 +1,13 @@
 {
   "tools": {
-    "dotnet": "3.1.200",
+    "dotnet": "5.0.100-preview.8.20417.9",
     "vs": {
       "version": "16.6"
     },
     "xcopy-msbuild": "16.6.5-alpha1"
   },
   "sdk": {
```

Comment:
In a similar roslyn PR, https://github.com/dotnet/roslyn/pull/46038/files, `"allowPrerelease": true,` was added. Is it necessary here?

Reply:
you are correct!